backend/src/users/achievements/entities/achievement.entity.ts

```diff
@@ -0,0 +1,10 @@
+import { Column, Entity, PrimaryGeneratedColumn } from "typeorm";
+
+@Entity("achievements")
+export class Achievement {
+	@PrimaryGeneratedColumn()
+	id: number;
+
+	@Column({unique: true})
```

Comment:
is this only protecting the achievements from doubles or will it also force users to have one achievement?